WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -238,10 +238,10 @@ def post_results
                 end
                 record_strs
               end.flatten
-              "#{uniqueName} #{record_strs.join(", ")}"
+              "#{uniqueName} #{record_strs.join(', ')}"
```

Comment:
I suppose the point of this rule was to deal with poor syntax highlighters? Either way, I agree, I'll remove this.